rules/detekt/src/main/kotlin/io/nlopez/compose/rules/detekt/RememberStateMissingCheck.kt

```diff
@@ -3,24 +3,20 @@
 package io.nlopez.compose.rules.detekt
 
 import io.gitlab.arturbosch.detekt.api.Config
-import io.gitlab.arturbosch.detekt.api.Debt
-import io.gitlab.arturbosch.detekt.api.Issue
-import io.gitlab.arturbosch.detekt.api.Severity
 import io.nlopez.compose.core.ComposeKtVisitor
 import io.nlopez.compose.rules.DetektRule
 import io.nlopez.compose.rules.RememberStateMissing
 
 class RememberStateMissingCheck(config: Config) :
-    DetektRule(config),
+    DetektRule(config, description),
     ComposeKtVisitor by RememberStateMissing() {
 
-    override val issue: Issue = Issue(
-        id = "RememberMissing",
-        severity = Severity.Defect,
-        description = """
+    override val ruleId = Id("RememberMissing")
+
+    private companion object {
+        private val description = """
             Using mutableStateOf/derivedStateOf in a @Composable function without it being inside of a remember function.
             If you don't remember the state instance, a new state instance will be created when the function is recomposed.
-        """.trimIndent(),
-        debt = Debt.FIVE_MINS,
-    )
+        """.trimIndent()
```

Comment:
Let's inline and simplify the text:
`mutableStateOf/derivedStateOf need to be remembered`